packages/client/src/bags/__fixtures__/putBagPromocodes.fixtures.ts

```diff
@@ -0,0 +1,17 @@
+import { rest, RestHandler } from 'msw';
+import type { BagPromoCodesInformation } from '../types';
+
+const path = '/api/commerce/v1/bags/:bagId/promocodes';
+
+const fixtures = {
+  success: (response: BagPromoCodesInformation): RestHandler =>
+    rest.put(path, async (_req, res, ctx) =>
```

Comment:
this async and on failure are unnecessary.

Reply:
Fixed 👍

Reply:
the async is still in the successful case.

Reply:
😓 fixed now